.github/workflows/build-push-stolostron.yaml

```diff
@@ -28,5 +28,5 @@ jobs:
         REPOSITORY=${{env.REPOSITORY}} \
```

Comment:
Optional: I don't think it's necessary to define an `env` if it's only used here once.

Reply:
good idea